cost-analyzer/values.yaml

```diff
@@ -304,6 +304,10 @@ global:
 ## Provide a full name override option for the chart.
 # fullnameOverride: ""
 
+## Configure Chart Labels
+# chartLabels:
+#   LabelKey: LabelValue
```

Comment:
```suggestion
## Provide additional labels for the chart.
# chartLabels:
#   app.kubernetes.io/name: kubecost-cost-analyzer
```

Hoping to provide a more descriptive comment and example. Let me know what you think!

Reply:
yes, it is better

Reply:
Ok, I'll commit them!